app/lib/contexts.json

```diff
@@ -0,0 +1,7 @@
+{
+  "stomachAche": {
+    "context": "stomach-ache",
```

Comment:
I think I prefer 'key' but think we should make the change when we add next add a context